src/main/resources/application.conf

```diff
@@ -100,6 +100,20 @@ constellation {
       aws-access-key = sys.env.get("CL_AWS_ACCESS_KEY").getOrElse("")
       aws-secret-key = sys.env.get("CL_AWS_SECRET_KEY").getOrElse("")
     }
+    aws-backup-a {
+      enabled = false
+      bucket-name = sys.env.get("CL_AWS_BUCKET_BACKUP_A").getOrElse("")
+      region = sys.env.get("CL_AWS_REGION_BACKUP_A").getOrElse("")
+      aws-access-key = sys.env.get("CL_AWS_ACCESS_KEY_BACKUP_A").getOrElse("")
+      aws-secret-key = sys.env.get("CL_AWS_SECRET_KEY_BACKUP_A").getOrElse("")
+    }
+    aws-backup-b {
+      enabled = false
+      bucket-name = sys.env.get("CL_AWS_BUCKET_BACKUP_B").getOrElse("")
+      region = sys.env.get("CL_AWS_REGION_BACKUP_B").getOrElse("")
+      aws-access-key = sys.env.get("CL_AWS_ACCESS_KEY_BACKUP_B").getOrElse("")
+      aws-secret-key = sys.env.get("CL_AWS_SECRET_KEY_BACKUP_B").getOrElse("")
+    }
```

Comment:
Or just:

```
aws {
  enabled = false
  bucket-names = sys.env.get("CL_AWS_BUCKETS")
}
```

As we have same key/secret/region for all the buckets. Doing `-a`,`-b`,`-c` manually doesn't scale anyway :P